.azure-pipelines-templates/test.yml

```diff
@@ -13,7 +13,7 @@ steps:
 # If a test fails, crudely upload nodes' logs for all tests
 - task: CopyFiles@2
   inputs:
-    contents: '**/?(out|err)'
+    contents: '**/?(*out|*err)'
```

Comment:
This doesn't seem to work just yet (see https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=5345&view=artifacts&type=publishedArtifacts).

Reply:
Fixed now: https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=5372&view=artifacts&type=publishedArtifacts